server/main.py

```diff
@@ -45,11 +48,7 @@ async def room(websocket: WebSocket) -> None:
     try:
         while True:
             event = await client.receive()
-            buggy, sender, event_data = await handler(event, room_code)
-
-            await manager.broadcast(event_data, room_code, sender=sender, buggy=buggy)
-    except WebSocketDisconnect as err:
-        await manager.broadcast(err.response, room_code, sender=client)
-        manager.disconnect(client, room_code)
-    except NotImplementedError as err:
-        await client.send(err.response)
```

Comment:
We need to tell the frontend when a client disconnects

Reply:
I see this logic has been implemented into the EventHandler class. Can resolve.

Reply:
We do that in the `EventHandler` with a sync event (to avoid problems with users having the same username).